src/ios/CameraPreview.m

```diff
@@ -738,52 +889,16 @@ - (void) invokeTakePicture:(CGFloat) width withHeight:(CGFloat) height withQuali
 
         CGImageRelease(finalImage); // release CGImageRef to remove memory leaks
 
-        CDVPluginResult *pluginResult;
-        if (self.storeToFile) {
-          NSData *data = UIImageJPEGRepresentation([UIImage imageWithCGImage:resultFinalImage], (CGFloat) quality);
-          NSString* filePath = [self getTempFilePath:@"jpg"];
-          NSError *err;
-         
-          if (![data writeToFile:filePath options:NSAtomicWrite error:&err]) {           
-            pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_IO_EXCEPTION messageAsString:[err localizedDescription]];
-          }
-          else {           
-            pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:[[NSURL fileURLWithPath:filePath] absoluteString]];
-          }
-        } else {
-          NSMutableArray *params = [[NSMutableArray alloc] init];
-          NSString *base64Image = [self getBase64Image:resultFinalImage withQuality:quality];
-          [params addObject:base64Image];
-          pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsArray:params];
-        }
+        NSString *base64Image = [self getBase64Image:resultFinalImage withQuality:quality];
 
         CGImageRelease(resultFinalImage); // release CGImageRef to remove memory leaks
 
+        [params addObject:base64Image];
+
+        CDVPluginResult *pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsArray:params];
         [pluginResult setKeepCallbackAsBool:true];
         [self.commandDelegate sendPluginResult:pluginResult callbackId:self.onPictureTakenHandlerId];
       }
     }];
 }
-
-- (NSString*)getTempDirectoryPath
-{
-  NSString* tmpPath = [NSTemporaryDirectory()stringByStandardizingPath];
-  return tmpPath;
-}
-
-- (NSString*)getTempFilePath:(NSString*)extension
-{
-    NSString* tmpPath = [self getTempDirectoryPath];
-    NSFileManager* fileMgr = [[NSFileManager alloc] init]; // recommended by Apple (vs [NSFileManager defaultManager]) to be threadsafe
-    NSString* filePath;
-
-    // generate unique file name
-    int i = 1;
-    do {
-        filePath = [NSString stringWithFormat:@"%@/%@%04d.%@", tmpPath, TMP_IMAGE_PREFIX, i++, extension];
-    } while ([fileMgr fileExistsAtPath:filePath]);
-    
-    return filePath;
-}
-
```

Comment:
Please re-add getTempDirectoryPath and getTempFilePath. I believe these are used for takeSnapshot.